mix.exs

```diff
@@ -16,7 +16,10 @@ defmodule Benchfella.Mixfile do
   end
 
   def application do
-    [applications: []]
+    [
+      applications: [],
```

Comment:
I'm not entirely sure that with newer Elixirs `applications:` is obligatory even.
Consider [a patch that [fixes similar issue in inch_ex](https://github.com/rrrene/inch_ex/pull/95/commits/1fe47e79b668f431f07b16db59b95eb4979fe652).

I would just leave `extra_applications: [:eex]`. I think it would be wise to also verify why does this supress the warning.

My hypothesis is that it has to do with the fact that newer Elixirs now know how to assemble minimal releases, but need a little help with figuring out which bits and bobs from the full Elixir distribution to include, but I'm not 💯% sure...

Reply:
I do not know I very simply did what the warning said 🤷

Reply:
but looking at the link you provided I would say they just did the same, no?

Reply:
Not quite. Note how inch_ex dropped `applications` key entirely.

Reply:
you can do that too of course I will not complain, is there any action on my side yet?

Reply:
@RobertDober I don't think so, now we wait for the maintaners, hah.